lib/cli/options.js

```diff
@@ -189,7 +190,7 @@ const loadPkgRc = (args = {}) => {
       }
     } catch (err) {
       if (args.package) {
-        throw new Error(`Unable to read/parse ${filepath}: ${err}`);
+        throw createUnparsableFileError(err, filepath);
```

Comment:
so, `createUnparsableFileError` wants a `string`, but here we're giving it an `Error`.  it will be coerced, of course, but we should probably avoid that

I think what we should do here is change it to something like:

```suggestion
        throw createUnparsableFileError(`Unable to read/parse ${filepath}: ${err}`, filepath);
```

my reasoning is:

1. Exceptions thrown from within parsers can be helpful, surprising, useless, etc.
1. Because of this, we should provide our own message, to provide the user some context
1. Additional information from the original error should also be provided, in case it's useful
1. Using the `reason` param for the "why" is likely going to be hidden from a user, since coercing an `Error` into a `string` shows only the `message` and formatted `stack` props.  so we should stuff everything we know about into the `message` prop.  (I think there is some sort of TC39 proposal for a nested error specification, which we should look into in the future since it applies to how we handle many errors)

Reply:
Makes sense. I have updated based on this.